portfolio.js

```diff
@@ -22,3 +33,105 @@ closeMenu.addEventListener('click', (e) => {
   menu.style.display = 'none';
   e.stopPropagation();
 });
+
+htmlMobileWorks.appendChild(mobileDynamic);
+mobileDynamic.innerHTML = "<main id='about' class='works'><main class='btw'><section class='section1 box'>  <header class='works_image'><header><h1>My Recent Works</h1></header><hr  class='horizontal_rule'/><div class='recent_works_image'><img src='img/myWorks.png' alt='image' /></div></header><main><h1>Multi-Post Stories</h1><p>A daily selection of privately personalized reads; no accounts orsign-ups required. has been the industry's standard dummy text eversince the 1500s, when an unknown printer took a standard dummy text.</p></main><section><ul><li><button class='ul_li'>CSS</button></li><li><button class='ul_li'>HTML</button></li><li><button class='ul_li'>BOOTSTRAP</button></li><li><button class='ul_li'>RUBY</button></li></ul></section><footer><button class='section1_button pop_btn'>See Project</div></button></footer></section></main></main>";
+
+for (let i = 0; i < 6; i += 1) {
+  mobileDynamic.innerHTML += ` <section class="section2 box background">
+  <section class="background_wrapper" style="background-image: url(${cardDetailsMobile[3].featuredImage})">
+      <main>
+          <h1>
+            ${cardDetailsMobile[0].name}
+          </h1>
+          <p>
+            ${cardDetailsMobile[1].description}
+          </p>
+        </main>
+        <section>
+          <ul>
+            <li>
+              <button class="ul_li">${cardDetailsMobile[2].technologies[0]}</button>
+            </li>
+            <li>
+              <button class="ul_li">${cardDetailsMobile[2].technologies[1]}</button>
+            </li>
+            <li>
+              <button class="ul_li">${cardDetailsMobile[2].technologies[2]}</button>
+            </li>
+          </ul>
+        </section>
+        <footer>
+          <button class="project pop_btn">See Project</button>
+        </footer>
+  </section>
+          </section>
+         `;
+}
+
```

Comment:
- [ ] Hi @BenjaminMatembe, I really commend your effort, You have really inserted alot of effort working on this milestone of the project, Currently i notice some un-aligned functionality in your `pop up modal` that makes it look different from the one we have in the Figma design.
I have recorded a video walking you through what you need to pay attention to and work on.
Also kindly check the `mobile view` and implement all the necessary changes i have pointed out.


Video Demonstration 👇 


https://user-images.githubusercontent.com/78634841/231532126-1ebe06bc-d0f9-4343-b0ba-86150d78549f.mp4

Reply:
Hi @brainconnect93  thank you for the feedback am really going to respond to it, however have noticed that the hover effect on the cards for the see project button to show so that when clicked it triggers a popup is not part of the requirements.
It is only two buttons in only two cards that show the see project button for the pop up.
kindly rectify more about this.